src/modules/deleteTodoTask.js

```diff
@@ -0,0 +1,6 @@
+function remove(index) {
+  const item = document.getElementById(index);
+  item.parentNode.removeChild(item);
+}
+
+export default remove;
```

Comment:
- [ ] You implemented a remove function for the DOM todos elements, however, this change doesn't get updated in your local storage so if I refresh the page, the todos I deleted appear again, which is not the behavior we are expecting. Please consider fixing it! 🤓 🔧